asset-swapper: log RFQ maker (un)blacklistings

feuGeneA opened the pull request:
## Description

Follow-up to https://github.com/0xProject/0x-monorepo/pull/2706
For re-use in new module `src/utils/rfq_maker_blacklist`:
-   promoted type `LogFunction` to module `src/types', and
-   promoted default logger function declarations to module `src/constants`.

## Testing instructions

Run the `rfq_maker_blacklist` unit tests and observe the logging output on the console.

## Types of changes

<!--- What types of changes does your code introduce? Uncomment all the bullets that apply: -->

<!-- * Bug fix (non-breaking change which fixes an issue) -->

* New feature (non-breaking change which adds functionality)

<!-- * Breaking change (fix or feature that would cause existing functionality to change) -->

feuGeneA committed:
asset-swapper: log RFQ maker (un)blacklistings

steveklebanoff commented on packages/asset-swapper/src/utils/rfq_maker_blacklist.ts:
```diff
@@ -18,6 +23,10 @@ export class RfqMakerBlacklist {
             if (this._makerTimeoutStreakLength[makerUrl] === this._timeoutStreakThreshold) {
                 this._makerBlacklistedUntilDate[makerUrl] =
                     Date.now() + this._blacklistDurationMinutes * constants.ONE_MINUTE_MS;
+                this.infoLogger(
```
super nit but could we assign `Date.now() + this._blacklistDurationMinutes * constants.ONE_MINUTE_MS;` to a variable and then use it on line 25 and this line ?

feuGeneA replied:
Addressed in https://github.com/0xProject/0x-monorepo/pull/2714/commits/84862f15c1cb6551748102a9a101b0a04dedc8e1

steveklebanoff reviewed (approved):
one nit but LGTM

feuGeneA committed:
Merge branch 'development' into feat/asset-swapper/rfq-maker-blacklist-logging

feuGeneA committed:
Store blacklistedUntil as local var, don't ref map

Addresses review comment https://github.com/0xProject/0x-monorepo/pull/2714#discussion_r497816501

feuGeneA committed:
Merge branch 'development' into feat/asset-swapper/rfq-maker-blacklist-logging

feuGeneA committed:
Blacklist for timeout === max, not just >

feuGeneA commented:
Verified on 0x-api staging:
![image](https://user-images.githubusercontent.com/7883777/94959450-33d82500-04bf-11eb-8aba-db34853ab05c.png)

feuGeneA commented:
and just to round it out:
![image](https://user-images.githubusercontent.com/7883777/94960657-44899a80-04c1-11eb-9d33-aec97096bfdd.png)
